lib/routes/account.js

```diff
@@ -405,52 +414,72 @@ module.exports = function (
           .then(sendVerifyLoginEmail)
           .then(recordSecurityEvent)
           .then(createResponse)
+          .catch(gateSigninUnblock)
           .done(reply, reply)
 
+        function checkIsForcedUnblock () {
+          let forced = config.signinUnblock && config.signinUnblock.enabled && config.signinUnblock.forcedEmails
```

Comment:
In config, you use [forcedEmailAddresses](https://github.com/mozilla/fxa-auth-server/pull/1471/files#diff-c8c5ee8e781f831e4927a392fa3565e6R517)